chart/compass/charts/connector/templates/certs-setup-job.yaml

```diff
@@ -15,8 +15,6 @@ spec:
         sidecar.istio.io/inject: “false”
     spec:
       serviceAccountName: {{ .Chart.Name }}-certs-setup-job
-      securityContext:
```

Comment:
I believe that this was added on purpose by @mwieczorek

Reply:
It was defined in two places, look at the line 30